lib/scala/searcher/src/main/scala/org/enso/searcher/sql/SqlSuggestionsRepo.scala

```diff
@@ -1129,7 +1129,17 @@ final class SqlSuggestionsRepo(val db: SqlDatabase)(implicit
           reexport         = reexport
         )
         row -> (firstArg +: args)
-      case Suggestion.Function(expr, module, name, args, returnType, scope) =>
```

Comment:
`SqlSuggestionsRepo` - may I know more about it? Where it is located? When it is created? What is its size?